### PR TITLE
Add object builder bindings to mobile

### PR DESCRIPTION
**Pull request by mcjcloud**
Adds motor functions in motor-mobile for interacting with the object builder

Note: Because of Cgo, the interface will be more explicit than when using Go directly. Each target language can use these functions to create an abstraction.

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1202528620406598/1202786913154580) by [Unito](https://www.unito.io)

### CODE REVIEW

File: bind/motor-mobile/object_builder.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+#include <stdlib.h>
+*/
+import "C"
+
+func NewObjectBuilder(n, d *C.char) error {
+	if instance == nil {
+		return errWalletNotExists
+	}
+
+	name := C.GoString(n)
```
Comment (joshLong145) [inline bind/motor-mobile/object_builder.go:26]: Can this panic or throw and error? we probably should check n and d for nil

File: bind/motor-mobile/object_builder.go [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+	objectBuilders[name] = builder
+	return nil
+}
+
+func SetObjectLabel(n, l *C.char) error {
+	if instance == nil {
+		return errWalletNotExists
+	}
+
+	name := C.GoString(n)
```
Comment (joshLong145) [inline bind/motor-mobile/object_builder.go:52]: same as a above on the nil check

**Review by joshLong145 — COMMENTED** [review]
Left a few comments inline on some examples. think we need more nil checks on the pointers since all this data will have to be passed by reference its important to check for nil refs.

File: bind/motor-mobile/object_builder.go (hunk 2 above)
Comment (mcjcloud) [reply bind/motor-mobile/object_builder.go:52]: Good catch, done.

File: pkg/motor/test_keys/psksnr1nu3qhdvz84kpfg604ys9pc8n4cvmgxyn84vyhw [hunk 3]
```diff
@@ -0,0 +1,2 @@
+C�
```
Comment (ntindle) [inline pkg/motor/test_keys/psksnr1nu3qhdvz84kpfg604ys9pc8n4cvmgxyn84vyhw]: Add to .gitignore

**Review by ntindle — APPROVED** [review]

**Review by joshLong145 — APPROVED** [review]

File: pkg/motor/test_keys/psksnr1nu3qhdvz84kpfg604ys9pc8n4cvmgxyn84vyhw (hunk 3 above)
Comment (mcjcloud) [reply pkg/motor/test_keys/psksnr1nu3qhdvz84kpfg604ys9pc8n4cvmgxyn84vyhw]: done

**Pull request merged**